packages/service/src/service.ts

```diff
@@ -6,7 +6,12 @@ import bodyParser from "body-parser";
 import session from "express-session";
 import http from "http";
 import upgradeWebsockets from "./websocket-service";
-import { createUser } from "./utils/users";
+import {
+  createUser,
```

Comment:
is this unused now? how do users get created?

Reply:
createUser will be used in a post request to create a new user. In `passport.use(...)`, createUser was swapped for validateUser since an account will already be created at this point.